src/kent/cli_server.py

```diff
@@ -25,5 +29,22 @@ def mock_get_interface_ip(family):
 serving.get_interface_ip = mock_get_interface_ip
```

Comment:
Because we're quelling the flask/werkzeug output, I can probably nix this mocking.